HW2/pom.xml

```diff
@@ -69,6 +69,14 @@
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <version>${project.parent.version}</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>9</source>
+                    <target>9</target>
```

Comment:
Are you sure, that you're specified a correct java version?

I believe you're using Java 11, Java 17 or Java 19.

Reply:
Yes, you are right, I am using java 11, but this change was proposed by Intellij idea when I used List.of(... in my code, a bit strange, looks like IDE bug